packages/utils/parcel-lsp/package.json

```diff
@@ -50,6 +50,6 @@
     "@typescript-eslint/parser": "^4.14.1",
     "eslint": "^7.19.0",
     "glob": "^7.1.6",
-    "typescript": "^4.6.4"
+    "typescript": "~4.7"
```

Comment:
These should be pinned too. If not they would use typescript@4.8 and having two ts versions installed will only cause problems. I would even recommend installing `typescript` as a dev dependency only in the root `package.json` and not in any of the packages. Would make things easier.